spec/test-outputs/www-integration.out.vcl

```diff
@@ -510,5 +518,21 @@ sub vcl_pass {
 }
 
 sub vcl_hash {
+  # include request method
+  set req.hash += req.request;
+
+  # support caching non-GET/HEAD requests
+  if (!(req.request == "GET" || req.request == "HEAD" || req.request == "FASTLYPURGE") && req.postbody) {
+    set req.hash += req.postbody;
+  }
+
+  # if handling GET/HEAD requests, fall back to normal
+  {
+    set req.hash += req.url;
+    set req.hash += req.http.host;
+    set req.hash += "#####GENERATION#####";
```

Comment:
What's this `#####GENERATION#####` for?

Reply:
Discussed on slack: it's from Fastly's VCL example (from having a conversation with Fastly support about this need; not from some generic documentation), we don't know what it does, but they provided it.